systers_portal/templates/community/community_search.html

```diff
@@ -4,7 +4,203 @@ <h1 align="center">Search for Communities</h1><br>
   <div class="md-form" >
     <input type="text" id="form1" class="form-control" placeholder="Search">
   </div>
+<div id="map-canvas" class="map-container mt20"></div>
 <hr>
+
+<script type="text/javascript" src="http://maps.google.com/maps/api/js"></script>
+<script type="text/javascript">
```

Comment:
You should create a separate JS file and link it here. Writing JS in the HTML file is not a very good practice.